lib/ui/shared/dialogs/schedule_dialog.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/view/dm_user/dm_user_viewmodel.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class ScheduleMess extends StatelessWidget {
```

Comment:
`class ScheduleMess extends ViewModelWidget<DmUserViewModel>`

This line will avoid you calling the viewmodel in your onPressed.

Now check line 14